packages/app-store/tandemvideo/api/callback.ts

```diff
@@ -74,5 +75,5 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     });
   }
 
-  res.redirect("/apps/installed");
+  res.redirect(getInstalledAppPath({ variant: "conferencing", slug: "tandem" }));
```

Comment:
This is to support redirecting to the correct category within the Installed Apps page once the app has been installed.